doc/ProductSearch.md

```diff
@@ -0,0 +1,491 @@
+# ProductSearch
+
+With the release of ViSenze's Catalog system, ViSearch Android SDK will now include the Product Search API suite - an additional set of APIs that ideally can achieve the following:
+
+- Select the right product image for indexing for best search performance
+- Aggregate search results on a product level instead of image level
+- Consistent data type in API response with Catalog’s schema
+
+> Current stable version: 1.5.0
```

Comment:
remove the version for now since we have not published. Latest is supposed to be 2.0